main/src/main/assets/full_licenses.html

```diff
@@ -236,7 +236,7 @@ <h3> Original SSLeay License</h3>
 <br> * The implementation was written so as to conform with Netscapes SSL.
 <br> * 
 <br> * This library is free for commercial and non-commercial use as long as
-<br> * the following conditions are aheared to.  The following conditions
+<br> * the following conditions are adhered to.  The following conditions
```

Comment:
These probably get dropped in the long run, in addition to being removed from this PR.